change/@ni-nimble-components-c588426a-6963-44b2-89a1-02697bed78d8.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Could you add a entry to nimble-components/CONTRIBUTING.md in the table under "Folder structure" that mentions our new `components` folder convention? It'd be good to mention that things in `components` are considered "internal".